src/turnkeyml/llm/README.md

```diff
@@ -0,0 +1,128 @@
+# Turnkey-LLM
```

Comment:
This PR is a major contribution! Some of this info definitely deservers to be part of the main README.md file of the repo.
(Could be a separate PR)

Reply:
Good idea!

Reply:
Added in https://github.com/onnx/turnkeyml/pull/225/commits/e7d7808b4b4135f6d76fbeb1f17ca6571b7cbaa2